app/models/user.rb

```diff
@@ -445,8 +445,6 @@ def erase!
   end
 
   def anonymise!
-    return if info_requests.none?
```

Comment:
> so that requests going forward are still censored

Ah, this isn't an intended behaviour (as requests would just get rejected by the authority as a name _must_ be present). 

Instead maybe we should disable anonymise button until the account is closed, in the same way that it must be closed before erasure?

Reply:
Okie dokie! I've removed the removal (hah), and I'm going to implement the disabling until the account is closed. If I can figure out a way to do it that makes sense, I might try and put some explanation of that and the clsoed-before-erased need on the page also, so it doesn't look like buttons are disabled at random.